docs/rulebook.md

```diff
@@ -0,0 +1,100 @@
+# Rulebook / constitution
+
+This is a small document that defines how this community is working.
+
+## Project role
+
+Provide a shared development environment and infrastructure for the Nix and
```

Comment:
Please change this doc to soft wrap, it's a bit annoying to try to keep hard wrap consistent and github markdown ignores it anyway.